pkg/ansible/operator/operator.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package operator
+
+import (
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/pkg/ansible/controller"
+	"github.com/operator-framework/operator-sdk/pkg/ansible/runner"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/sirupsen/logrus"
+)
+
+func RunSDK(done chan error, mgr manager.Manager, namespace string) {
```

Comment:
I wonder if `RunSDK` can have a better name. What if `RunAnsibleOperator` or `Run` (using the namespace as an identifier will be a more descriptive name. 

I also think that we need to make sure that people are aware that this is a blocking function that will call start on the manager. I would really like to see a godoc describing what it is doing (getting watches file, adding controllers to the manager, running the manager).